<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZFieldDLg</class>
 <widget class="QDialog" name="ZFieldDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fieldlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeLEd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeStrLEd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="FieldNameLEd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DiscardBTn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptBTn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QWidget" name="ZTypeListWDg" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tooltip</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TooltipLEd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="SizeFRm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Header size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HeaderSizeLEd">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Natural size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NaturalLEd">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Universal size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UniversalLEd">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CapacityLEd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="KeyEligibleCHk">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="defines wether field may be part of a key or not"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="defines wether field may be part of a key or not"/>
    </property>
    <property name="text">
     <string>Key eligible</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateArrayBTn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create array from capacity</string>
    </property>
   </widget>
   <widget class="QLabel" name="KeyEligibleLBl">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="defines wether field may be part of a key or not"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="CommentLBl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QFrame" name="HashcodeFRm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ValidateHashBTn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Validate hashcode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HashcodeLEd">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ComputeBTn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute hashcode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hashcode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FieldNameLEd</tabstop>
  <tabstop>AcceptBTn</tabstop>
  <tabstop>DiscardBTn</tabstop>
  <tabstop>ZTypeStrLEd</tabstop>
  <tabstop>ZTypeLEd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
